common/dllserver/CMakeLists.txt

```diff
@@ -38,7 +38,8 @@ include_directories (
          ./../../rtl/eclrtl 
          ./../../system/include 
          ./../../dali/base 
-         ./../../system/jlib 
+         ./../../system/jlib
+         ./../../ecl/hql
```

Comment:
This dll shouldn't really have a dependency on hql.
I think the cleanest solution is to move getResourceFromJfrog into jlib.  Either into a new file, or added to something like jutil.hpp/cpp

Reply:
Moved getResourceFromJfrog to jutil.